torch/onnx/symbolic_helper.py

```diff
@@ -612,7 +611,7 @@ def _onnx_unsupported(op_name: str, value: Optional[_C.Value] = None) -> NoRetur
         f"Please feel free to request support or submit a pull request "
         f"on PyTorch GitHub: {_constants.PYTORCH_GITHUB_ISSUES_URL}"
     )
-    if isinstance(value, _C.Value):
+    if _is_value(value):
         raise errors.SymbolicValueError(
             message,
             value,
```

Comment:
But looks like this version of mypy does not support it

Reply:
do you know which mypy version introduced this? CI seems to be using mypy 0.960

Reply:
Not sure. I installed the latest version locally but still got the error. It should have been supported since last year https://github.com/python/mypy/pull/9865

Reply:
Is this fixed?

Reply:
not sure 🤔